backend/api/views/message.py

```diff
@@ -1,11 +1,149 @@
-from flask import Blueprint, request
-from api.models.Message import Message
+from flask import Blueprint, request, current_app
+from api.models import (
+    Message,
+    FieldPartner,
+    PortfolioManager,
+    Document,
+    DocumentClass,
+    db,
+)
+from flask_mail import Message as Flask_Message
+from flask_mail import Mail
 from api.core import create_response, serialize_list, logger
+from enum import Enum
+import os
 
 message = Blueprint("message", __name__)
 
 
+class MessageType(Enum):
+    NEW_DOC = 0
+    REVIEWED_DOC = 1
+    UPLOADED_DOC = 2
+
+
 @message.route("/messages", methods=["GET"])
 def get_messages():
     messages = Message.query.all()
     return create_response(data={"messages": serialize_list(messages)})
+
+
+@message.route("/messages/fp/<fp_id>", methods=["GET"])
```

Comment:
ideally this should be done using a query string but thats one of the changes in my refactor pr so this is fine for now